bears/python/PyUnusedCodeBear.py

```diff
@@ -15,7 +15,7 @@ class PyUnusedCodeBear(LocalBear):
     CAN_DETECT = {'Unused Code'}
 
     def run(self, filename, file,
-            remove_all_unused_imports: bool=False,
+            remove_all_unused_imports: bool=True,
             remove_unused_variables: bool=True):
         """
         Detects unused code. By default this functionality is limited to:
```

Comment:
Was actually done by https://github.com/coala/coala-bears/pull/1601 in this case